tests/check_merge_conflict_test.py

```diff
@@ -64,6 +64,22 @@ def f1_is_a_conflict_file(tmpdir):
         yield repo2
 
 
+@pytest.fixture
+def worktree_f1_is_a_conflict_file(f1_is_a_conflict_file):
+    worktree = f1_is_a_conflict_file.dirpath().join('worktree')
+    cmd_output('git', 'worktree', 'add', str(worktree))
+    with worktree.as_cwd():
+        cmd_output(
+            'git', 'pull', '--no-rebase', 'origin', 'master', retcode=None,
+        )
+        assert os.path.exists(
+            f1_is_a_conflict_file.join(
+                '.git', 'worktrees', 'worktree', 'MERGE_MSG',
+            ),
+        )
+        yield worktree
```

Comment:
I meant that you would inline this directly in your test above the call to the other test

no need to make a fixture if only one thing is using it